Add firestore db

maggiemay-afk opened the pull request:
Installed more Firebase packages for Firestore DB
Added helpful Firestore DB links to Read Me file
Reworked the firebase.ts file to initialize our Firestore DB
Installed ts-node to run scripts 
Added a 'mapping' object (at: ambiance/index.ts) to help with scripts / seeding DB
Created / finalized scripts and used them to seed database

@hermanator608 take a look at the Firebase console when you get a chance to see all the data imported

hermanator608 commented on README.md:
```diff
@@ -57,3 +57,7 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 
 To learn React, check out the [React documentation](https://reactjs.org/).
+
+## Useful links
+https://firebase.google.com/docs/firestore/quickstart#node.js 
+https://firebase.google.com/docs/web/setup 
```
TY! Would be awesome to do a PR specifically to update the README.md. Could have an entire section dedicated to Firestore DB

hermanator608 commented on scripts/seedDatabase.ts:
```diff
@@ -0,0 +1,55 @@
+const app = initializeApp(firebaseConfig);
+const db = getFirestore(app);
+
+//addChannels();
+//addWarcraft();
```
So since these are both commented out, if we ran this script nothing would happen but if I didnt know about this script I could assume that it might have properly ran the seed. Can we make it so we log what is seeded and enable these. For example:
```
Seeding Channels...

...

Channels Seeded!
```

hermanator608 replied:
Could also add some sort of script params to do this:
```
yarn seedDatabase channels
yarn seedDatabase ambiance
```

hermanator608 commented on scripts/seedDatabase.ts:
```diff
@@ -0,0 +1,55 @@
+}
+
+
+async function addWarcraft() {
```
This name doesnt really make sense as its all ambiance categories and not just Warcraft

hermanator608 reviewed (commented):
Great work - few thoughts and suggestions

maggiemay-afk commented:
Definitely can update the README in a later PR. 
I added a switch to the scripts file for command line arguments ("channels" and "ambiance").  
Added logs to those 2 functions as well, like you suggested.
And forgot to update the function name after I was done testing - that's done as well 

Thanks! @hermanator608
